### PR TITLE
search 7.x | elasticsearch connector config reference | LRDOCS-9431

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-9431
cc @lipusz, @jrhoun
Rich, JR, we made use of the HTML `details` tag, so that our enormous table can take advantage of collapsible property descriptions--it's never the case that you need to see them all at once--you're probably scanning this material to find just the property you need.

### CODE REVIEW

File: docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/elasticsearch-connector-settings.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The connection to Elasticsearch is primarily defined in the _Elasticsearch 6/7_
 | --------------- | -------------------------------------- | 
 | Liferay 7.2.x<br />Elasticsearch 6.x  | Elasticsearch 6<br />`com.liferay.portal.search.elasticsearch6.configuration.ElasticsearchConfiguration.config` |
 | Liferay 7.2.x<br />Elasticsearch 7.x  | Elasticsearch 7<br />`com.liferay.portal.search.elasticsearch7.configuration.ElasticsearchConfiguration.config` |
-| Liferay 7.3.x<br />Elasticsearch 7.x  | Elasticsearch 7<br />`com.liferay.portal.search.elasticsearch7.configuration.ElasticsearchConfiguration.config`<br /><br />Elasticsearch Connections (factory)<br />`com.liferay.portal.search.elasticsearch7.configuration.ElasticsearchConnectionConfiguration-[key].config` |
+| Liferay 7.3.x<br />Elasticsearch 7.x  | Elasticsearch 7<br />`com.liferay.portal.search.elasticsearch7.configuration.ElasticsearchConfiguration.config`<br /><br />Elasticsearch Connections (factory)<br />`com.liferay.portal.search.elasticsearch7.configuration.ElasticsearchConnectionConfiguration-[connectionId].config` |
```
Comment (lipusz) [inline docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/elasticsearch-connector-settings.md:13]: :star_struck:

**Comment by rbohl** [comment]
This is how it looks rendered in the site:
details-expandable

**Comment by lipusz** [comment]
Hello @sez11a ,

Do you have any updates on this?

Thanks,
Tibor

**Comment by sez11a** [comment]
Superseded by https://github.com/jrhoun/liferay-learn/pull/1135

**Pull request closed without merging**